LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright 2015 MongoDB, Inc.
+Copyright 2020 MongoDB, Inc.
```

Comment:
The Apache license doesn't specify how to handle years. The GPL suggests a range https://www.gnu.org/licenses/gpl-howto.html, which is what we do elsewhere https://github.com/mongodb/mongo-ruby-driver/blob/master/LICENSE#L178.